[1.31.0] - Snapshots - finalize docs (#401)
replicatedhq/kots.io: content/kotsadm/snapshots/overview.md

---
date: 2021-01-19
linktitle: "Overview"
weight: 8
title: Overview
---

An important part of the lifecycle of an application is backup and restore. The Admin Console can be used to create and manage your storage destination and schedule, and to perform and monitor the backup and restore process. This feature is only available for licenses that have the Allow Snapshots feature enabled.

Currently, there are two types of snapshots:
  * Full Snapshots (Instance)
  * Partial Snapshots (Application)
  
Snapshots are useful for rollback and disaster recovery scenarios. They are not intended to be used for application migration scenarios. 

## Full Snapshots (Recommended)

Full snapshots back up the Admin Console and all application data.
They can be used for full Disaster Recovery; by restoring over the same instance, or into a new cluster.

There are two ways to create a full snapshot. First, make sure that your license has the snapshots feature enabled, then:

1. Using the KOTS CLI [backup](/kots-cli/backup/) command.
2. Using the Admin Console (check screenshot below).

![Instance Backup UI](/images/snapshot-instance-backup.png)

> After a full snapshot is created you can restore it by clicking restore icon on a full snapshot row (check screenshot below).

Author: ```suggestion
After a Full Snapshot has been created, you can get a command to do a restore by clicking the restore icon (check screenshot below).
```

> There are two available options for restore. You can do a full restore of the application, admin console, and databases or you can do a partial restore of just your application and its data.

Author: ```suggestion
There are two available options for doing a restore. You can either do a full restore which will restore the Admin Console and the application with all of its data, or you can choose to do a partial restore of just your application and its data.
```
